### ISSUE TITLE
Single d deletes line under cursor (Neovim)

### ISSUE THREAD

**Issue by joereynolds** [markonm/traces.vim#4]
Pressing d once deletes the line under the cursor instead of dd.

To reproduce:

I'm using Neovim 0.2. and vimplug as the plugin manager

This is the init.vim (vimrc) required to reproduce the issue
```
call plug#begin()
    Plug 'xtal8/traces.vim'
call plug#end()
```

I have noticed a similar bug logged with deoplete
https://github.com/Shougo/deoplete.nvim/issues/460

Thanks :)

**Comment by markonm** [comment]
It's a problem with Neovim's timers. It can be reproduced with this snippet:

```vim
func! g:Foo(...)
  " do nothing
endfunc
call timer_start(15,function('g:Foo'),{'repeat':-1})
```
It's seems it's fixed in Neovim 0.2.1-dev. There's no other solution apart from increasing Neovim's requirements. Thank's for letting me know.

**Comment by markonm** [comment]
I updated Neovim version requirements. https://github.com/xtal8/traces.vim/commit/fab8e99ee87cb140b51d06ac5526aced7d063878
